### DISCUSSION TITLE
Can not check faster than 5 seconds.

### DISCUSSION THREAD

**Discussion by Vodkaice35** [jef/streetmerchant#1828] [Help]
I have been trying to check only one store with between 1 to 2 seconds.

I have setup dotenv
PAGE_SLEEP_MIN=1000
PAGE_SLEEP_MAX=2000

I also tried with headless false to check how it's doing, as soon as it starts, it opens only 1 link at a time and in this link it waits 5 seconds before going to next link. So this is SO SO SLOW.

How can I fix it ? I want to check all links simultaneously. Because 1 page at a time when headless false is so slow. I think this is also works like that with headless true. If there is no way to do this at least I want to check pages withing 1 second.

Any help ?

![1](https://user-images.githubusercontent.com/78175755/106194156-84d2fe00-61bf-11eb-96f2-3239c70d8858.JPG)

**Comment by wofnull** [comment]
This is set in place to prevent Amazon ( and other sites ) from triggering the anti bot mechanic.
Each Website has its own cooldown, checks at the same time are not possible ( if checking only one store )

**Reply by Vodkaice35** [reply]
> This is set in place to prevent Amazon ( and other sites ) from triggering the anti bot mechanic.
> Each Website has its own cooldown, checks at the same time are not possible ( if checking only one store )

How can I bypass this ? I have no problem with captcha's with my local amazon store, hence it wont be a problem.
